From 2ff3fc73c81e0170c46683589d9ad705f3a5ad45 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Dan=20Hor=C3=A1k?= <dan@danny.cz>
Date: Fri, 2 Mar 2012 11:41:37 +0100
Subject: [PATCH] install ctcm network module on s390

The ctcm module is not loaded automagically because it doesn't pass the
"ether_type_trans" test in 40net/module-setup.sh, so load it explicitly.
---
 modules.d/95znet/module-setup.sh |    4 ++++
 1 file changed, 4 insertions(+)

diff --git a/modules.d/95znet/module-setup.sh b/modules.d/95znet/module-setup.sh
index fbb79aa..b354947 100755
--- a/modules.d/95znet/module-setup.sh
+++ b/modules.d/95znet/module-setup.sh
@@ -13,6 +13,10 @@ depends() {
     return 0
 }
 
+installkernel() {
+    instmods ctcm
+}
+
 install() {
     inst_hook cmdline 30 "$moddir/parse-ccw.sh"
     inst /lib/udev/ccw_init
